project/.travis.yml.twig

```diff
@@ -64,6 +64,10 @@ matrix:
 {% endfor %}
     - php: '{{ php|last }}'
       env: SYMFONY_DEPRECATIONS_HELPER=0
+{% if 'mongodb' in services %}
```

Comment:
I don't get how [this](https://github.com/sonata-project/dev-kit/blob/8e1af1e5d91ed9f3b8fa7ca0eec26417655ba12b/config/dev-kit.yml#L45) should be used